### PR TITLE
Update HTTP Cache Partitioning article

### PR DESCRIPTION
**Pull request by agektmr**
What's changed, or what was fixed?
- Updates HTTP Cache Partitioning article

**Target Live Date:** 2020-10-24

- [ ] This has been reviewed and approved by (NAME)
- [x] I have run `npm test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele @shivanigithub

### CODE REVIEW

**Comment by agektmr** [comment]
@kaycebasques @jpmedley May I ask one of you to review this pull request? Thanks.

File: src/content/en/updates/2020/10/http-cache-partitioning.md [hunk 1]
```diff
@@ -205,6 +205,31 @@ occurs.
...
+2. Specify where to save the log file on your computer.
+3. Browse the web on Chrome for a minute.
+4. Go back to `chrome://net-export/` and press **Stop Logging**.
+5. Go to `https://netlog-viewer.appspot.com/#import`.
+6. Press **Choose File** and pass the log file you saved.
+7. You will see the output of the log file.
+
+On the same page, find `SplitCacheByNetworkIsolationKey`. If it is followed by
+`Experiment_[****]`, HTTP Cache Partitioning is enabled on your Chrome. If it is
+followed by `Control_[****]` or `Default_[****]`, not enabled.
```
Comment (shivanigithub) [inline src/content/en/updates/2020/10/http-cache-partitioning.md:223]: nit: "not enabled" to be replaced with "it is not enabled"

**Review by shivanigithub — COMMENTED** [review]
The changes lgtm, thanks!

File: src/content/en/updates/2020/10/http-cache-partitioning.md [hunk 2]
```diff
@@ -205,6 +205,31 @@ occurs.
 
 ## FAQs
 
+### Is it already enabled on my Chrome? How can I check?
+
+The feature is being rolled out through late 2020. To check whether your Chrome instance is
+already in effect, follow the instructions below:
```
Comment (jpmedley) [inline src/content/en/updates/2020/10/http-cache-partitioning.md:211]:
```suggestion
The feature is being rolled out through late 2020. To check whether your Chrome instance
already supports it:
```

File: src/content/en/updates/2020/10/http-cache-partitioning.md [hunk 3]
```diff
@@ -205,6 +205,31 @@ occurs.
...
+
+1. Open `chrome://net-export/` and press **Start Logging to Disk**.
+2. Specify where to save the log file on your computer.
+3. Browse the web on Chrome for a minute.
+4. Go back to `chrome://net-export/` and press **Stop Logging**.
+5. Go to `https://netlog-viewer.appspot.com/#import`.
+6. Press **Choose File** and pass the log file you saved.
+7. You will see the output of the log file.
```
Comment (jpmedley) [inline src/content/en/updates/2020/10/http-cache-partitioning.md:219]:
```suggestion
6. Press **Choose File** and pass the log file you saved. 

You will see the output of the log file.
```

Comment (jpmedley) [reply src/content/en/updates/2020/10/http-cache-partitioning.md:219]: The best practice is that steps only begin with a user action. Responses to steps are tacked at the end of the step. The result of the whole procedure is in a separate paragraph at the end.

File: src/content/en/updates/2020/10/http-cache-partitioning.md (hunk 1 above)
Comment (jpmedley) [inline src/content/en/updates/2020/10/http-cache-partitioning.md:223]:
```suggestion
followed by `Control_[****]` or `Default_[****]`, it is not enabled.
```

File: src/content/en/updates/2020/10/http-cache-partitioning.md [hunk 4]
```diff
@@ -205,6 +205,31 @@ occurs.
...
+
+On the same page, find `SplitCacheByNetworkIsolationKey`. If it is followed by
+`Experiment_[****]`, HTTP Cache Partitioning is enabled on your Chrome. If it is
```
Comment (jpmedley) [inline src/content/en/updates/2020/10/http-cache-partitioning.md:222]:
```suggestion
`Experiment_[****]`, HTTP Cache partitioning is enabled on your Chrome. If it is
```

File: src/content/en/updates/2020/10/http-cache-partitioning.md [hunk 5]
```diff
@@ -205,6 +205,31 @@ occurs.
...
+followed by `Control_[****]` or `Default_[****]`, not enabled.
+
+### How can I test HTTP Cache Partitioning on my Chrome?
```
Comment (jpmedley) [inline src/content/en/updates/2020/10/http-cache-partitioning.md:225]:
```suggestion
### How can I test HTTP Cache partitioning on my Chrome?
```

File: src/content/en/updates/2020/10/http-cache-partitioning.md [hunk 6]
```diff
@@ -205,6 +205,31 @@ occurs.
...
+
+To test HTTP Cache Partitioning on your Chrome, you need to launch Chrome with a
```
Comment (jpmedley) [inline src/content/en/updates/2020/10/http-cache-partitioning.md:227]:
```suggestion
To test HTTP Cache partitioning on your Chrome, you need to launch Chrome with a
```

File: src/content/en/updates/2020/10/http-cache-partitioning.md [hunk 7]
```diff
@@ -205,6 +205,31 @@ occurs.
...
+command line flag `--enable-features=SplitCacheByNetworkIsolationKey`. Follow
```
Comment (jpmedley) [inline src/content/en/updates/2020/10/http-cache-partitioning.md:228]:
```suggestion
command line flag: `--enable-features=SplitCacheByNetworkIsolationKey`. Follow
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Comment by agektmr** [comment]
Thanks for the reviews. I'll merge.

**Pull request merged**